src/__tests__/DashboardView/DashboardView.js

```diff
@@ -2,23 +2,30 @@ import React from 'react';
 import DashboardView from '../../DashboardView/DashboardView.js';
 import renderer from 'react-test-renderer';
 import TestData from '../testData/products.js'
+import ProductFamilyData from '../testData/frontend_productFamilies'
 let instance, component, notificationService;
+const fetchProducts = jest.fn();
+const fetchProductFamilies = jest.fn();3
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Reformat the code to have only one statement per line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=javascript%3AOneStatementPerLine)